Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-5331a.dfy

```diff
@@ -0,0 +1,9 @@
+// RUN: %baredafny verify %args --type-system-refresh "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype S  = S(
+    n:nat
+) {
+    function f():S { this }
+    function g():S { this.(n := 0).f() }
```

Comment:
This file contains the failing test in Issue #5365, right? If so, should the name of the file reflect this?